heroku/react-hk-components
# Removed Unused Enzyme Dependencies

## Why the change?
[GUS Ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001LkwwiYAB/view)
In doing some initial work to get us compatible with React 18, I noticed that we had some dependencies on Enzyme that we were not using. Enzyme will never be compatible with React 18 it looks like, so it does not make sense to keep this around. 

## How to verify the change?
+ visit the PR app 
+ Make sure it works like in production 
+ The tests pass
